src/app/app.jsx

```diff
@@ -11,11 +11,11 @@ import { routerReducer, syncHistoryWithStore } from 'react-router-redux'
 
 import * as reducers from '../reducers/reducers.js'
 
-import { GetBusStopsGeoState } from '../containers/getBusStopsGeoState.js';
+import GetBusStopsGeoState from '../containers/getBusStopsGeoState.js';
```

Comment:
is need to in containers?

container is a js file I think that is only problem in jsx files with this ES6 syntax